e2e/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const PRODUCT_NAME = process.env.PLAYWRIGHT_PRODUCT_NAME;
+export const BASE_URL = process.env.PLAYWRIGHT_TEST_BASE_URL;
+export const HOMEPAGE_ENDPOINT = '/';
+export const WOMEN_COLLECTIONS_ENDPOINT = '/collections/women';
```

Comment:
`/collections/` is part of the app, but `women` needs to be user-configurable (e.g., `PLAYWRIGHT_COLLECTION_HANDLE`).